tests/Custom/SessionResetTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Mink\WebdriverClassDriver\Tests\Custom;
+
+use Behat\Mink\Tests\Driver\TestCase;
+
+class SessionResetTest extends TestCase
+{
+    /**
+     * @dataProvider initialWindowNameDataProvider
+     */
+    public function testSessionResetClosesWindows(?string $initialWindowName): void
```

Comment:
@stof Would it make sense to put this test in driver-testsuite?

Reply:
I've added this test into the driver's test suite instead of the Mink driver test suite because the window closing behavior upon session reset is non-standard or advertised anywhere.